client/src/components/pages/Profile/index.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Tabs, Icon } from "antd";
+import PropTypes from "prop-types";
+
+import PersonalInfoForm from "components/utils/ProfilePersonal";
+import BusinessForm from "components/utils/businessForm";
+import ProfilePic from "components/pages/Profile/ProfilePic";
+import "./style.css";
+
+const { TabPane } = Tabs;
+
+export default function ProfilePage({
```

Comment:
it should be a class, since we will use ComponentDidMount to get the user info from the database.